src/app/style/abstract/_icons.scss

```diff
@@ -473,3 +473,66 @@
     margin: 0 4px;
     width: 20px;
 }
+
+@mixin share-button {
+    $share-button-stroke-width: calc(var(--header-icon-stroke-width) + 1px);
```

Comment:
Share button is not getting value on mobile from navigationTabs component (2px) and on desktop it should be 2px also.